mappings/net/minecraft/stat/ServerStatHandler.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS xs net/minecraft/stat/ServerStatHandler
+	FIELD b LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD c server Lnet/minecraft/server/MinecraftServer;
+	FIELD d statFile Ljava/io/File;
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;Ljava/io/File;)V
+		ARG 1 server
+		ARG 2 statsFile
+	METHOD a saveStats ()V
```

Comment:
If it's a ServerStatHandler, do we need the verbose *Stat/*Stats?

Reply:
What about "statsToString"? "toString" would conflict with the default java method.

Reply:
"asString"? Depends on the context.

Reply:
They're talking about another method